src/cloudai/systems/slurm/slurm_system.py

```diff
@@ -448,7 +448,14 @@ def is_job_running(self, job_id: int, retry_threshold: int = 3) -> bool:
 
             if "Socket timed out" in stderr or "slurm_load_jobs error" in stderr:
                 retry_count += 1
-                logging.warning(f"Transient error encountered. Retrying... " f"({retry_count}/{retry_threshold})")
+                logging.warning(
+                    f"An error occurred while querying the job status. Retrying... ({retry_count}/{retry_threshold}). "
+                    "CloudAI uses Slurm commands by default to check the job status. The Slurm daemon can become "
+                    "overloaded and unresponsive, causing this error message. CloudAI retries the command multiple "
+                    f"times, with a maximum of {retry_threshold} attempts. There is no action required from the user "
+                    "for this warning. Please ensure that the Slurm daemon is running and responsive. If the problem "
+                    "persists, consider increasing the retry threshold."
```

Comment:
How can user increase the threshold?

Reply:
There is no external interface. Therefore, we may need to update the message to avoid misleading users.

Reply:
Please let me know your thoughts, @amaslenn .

Reply:
Yes, let's remove the part about increasing for now. Not sure we need to add an external control for that.

Reply:
Updated, @amaslenn